frontend/src/components/annotation/annotation_ui_factory.vue

```diff
@@ -1021,9 +1034,36 @@ export default Vue.extend({
                 return {size: 100 / this.annotation_ui_context.panel_settings.rows}
             })
             this.rows_panes_size = default_pane_sizes_rows
-            this.recalculate_pane_rows_dimensions(default_pane_sizes_rows)
+            this.calc_init_rows_dimensions(default_pane_sizes_rows)
```

Comment:
Hey @anthony-sarkis , let me know if we want to revert the current layout logic that stacks panes on top of each other that we enabled for the demo.